scripts/openstack-quickstart-demosetup

```diff
@@ -1138,7 +1138,7 @@ if [ "x$with_tempest" = "xyes" -a -e /etc/tempest/tempest.conf ]; then
     crudini --set /etc/tempest/tempest.conf compute fixed_network_name fixed
     crudini --set /etc/tempest/tempest.conf compute network_for_ssh ext
     crudini --set /etc/tempest/tempest.conf compute-admin username admin
-    crudini --set /etc/tempest/tempest.conf compute-admin tenant_name admin
+    crudini --set /etc/tempest/tempest.conf compute-admin project_name admin
```

Comment:
`compute-admin` group has been [dropped](https://opendev.org/openstack/tempest/commit/daadcdbec6c8fec4a5e42b4924f9b8480ac222e5) so we can probably remove all these settings safely.

Reply:
Done